curriculum/challenges/chinese/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-8
 
 # --description--
 
-HTML <dfn>属性</dfn>是写在一个元素的开始标签中的特殊词汇，它们用于控制这个元素的行为。 在 `img` 元素中的 `src` 属性明确了一个图片的 URL（图片所在的位置）。 这是一个使用了 `src` 属性的 `img` 元素：`<img src="https://www.your-image-source.com/your-image.jpg">`。
+HTML <dfn>属性</dfn>是写在一个元素的开始标签中的特殊词汇，它们用于控制这个元素的行为。 在 `img` 元素中的 `src` 属性明确了一个图片的 URL（图片所在的位置）。 这是一个使用了 `src` 属性的 `img` 元素：`<img src="https://www.host-website.com/the-image.jpg">`。
```

Comment:
please change only the english file, not the chinese one

Reply:
I wasn't able to find the English one. Is it possibly translated to English from this when users are on the website? If you find the correct file, let me know.

Reply:
Here's English file https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc24073f86c76b9248c6ebb.md

Reply:
Oh perfect I see how you chose the English folder! Thank you so much. Now I know for the future! How do I remove that previous commit and the one that is unrelated to this pull request now that I will only be updating the English file?